packages/dx-react-chart-bootstrap4/src/templates/axis/label.jsx

```diff
@@ -1,19 +1,21 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
 
 export class Label extends React.PureComponent {
   render() {
     const {
-      text, x, y, dominantBaseline, textAnchor,
+      text, x, y, dominantBaseline, textAnchor, className, ...restProps
     } = this.props;
 
     return (
       <text
         dominantBaseline={dominantBaseline}
         textAnchor={textAnchor}
-        key={text}
         x={x}
         y={y}
+        className={classNames('dx-c-bs4-fill-current-color', className)}
```

Comment:
Can't find the `dx-c-bs4-fill-current-color` class in css.

Reply:
It is [here](https://github.com/DevExpress/devextreme-reactive/pull/1046/files#diff-2db61a31060483590a78fd47ef3ed6e8)